src/coreclr-debug/activate.ts

```diff
@@ -113,3 +110,46 @@ function showDotnetToolsWarning(message: string) : void
             });
     }
 }
+
+interface AdapterExecutableCommand {
+    command: string;
+}
+
+// The default extension manifest calls this command as the adapterExecutableCommand
+// If the debugger components have not finished downloading, the proxy displays an error message to the user
+// Else it will launch the debug adapter
+export async function registerAdapterExecutionCommand(channel: vscode.OutputChannel): Promise<AdapterExecutableCommand> {
```

Comment:
nit: It seems like a better name for this method is 'getAdapterExecutionCommand` since it doesn't seem like it registers anything.